src/elements/content-preview/preview-header/PreviewHeader.scss

```diff
@@ -89,6 +89,12 @@
 .bcpr-PreviewHeader-controls {
     display: flex;
 
+    .bcpr-PreviewHeader-contentAnswers {
+        display: flex;
+        align-items: center;
+        padding: 0 5px;
+    }
```

Comment:
there is vertical alignment issue when used alongside OpenWith

**Before**
<img width="260" alt="Screenshot 2024-10-18 at 11 21 10 AM" src="https://github.com/user-attachments/assets/7e6ce629-100b-4800-820b-86f6a8723c02">

**After**
<img width="260" alt="Screenshot 2024-10-18 at 11 20 34 AM" src="https://github.com/user-attachments/assets/f9024ad8-f385-446c-83c4-e22e9244f28c">

Reply:
🤨